tools/lambda-promtail/lambda-promtail/s3.go

```diff
@@ -51,7 +52,7 @@ func getS3Object(ctx context.Context, labels map[string]string) (io.ReadCloser,
 		s3Client = s3.NewFromConfig(cfg)
 		s3Clients[labels["bucket_region"]] = s3Client
 	}
-
+	fmt.Println("fetching", labels["key"])
```

Comment:
did you mean to include this println?

Reply:
I used it for debugging purposes and ended up leaving it there since it's a useful piece of information (it helps immediately know what event/file is being processed) and because it prints only 1 line per lambda invocation (so it's not like it's going to cost a lot in Cloudwatch logs).

As the code is growing maybe we should have a more structured approach.
Maybe we could remove this line and open up another MR to use the standard logger (and set the LOG_LEVEL or DEBUG using an environment variable)? WDYT?

Reply:
I'm fine with keeping as is or implementing something more structured.

Reply:
Okay I implemented the logger in my latest commit, I went with go-kit since it's the one used in the main project